src/plugins/content_management/server/core/types.ts

```diff
@@ -30,9 +31,60 @@ export interface ContentStorage {
   delete(ctx: StorageContext, id: string, options: unknown): Promise<any>;
 }
 
+export interface RpcSchemas {
+  get?: {
```

Comment:
Why some of them are optional, but some of them are required?

Reply:
Because when creating a content we enforce a schema to be defined for the data before sending it to the DB layer as best practice. But to fetch there is nothing to enforce.